tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -124,6 +125,33 @@ var _ = Describe("odo devfile create command tests", func() {
 			values := gjson.GetMany(output, "kind", "metadata.name", "status.state")
 			Expect(helper.GjsonMatcher(values, []string{"Component", "nodejs", "Unknown"})).To(Equal(true))
 		})
+
+		It("should successfully create the devfile component and show json output for a unreachable cluster", func() {
+
+			path := os.Getenv("KUBECONFIG")
+
+			// read the contents from the kubeconfig and replace the server entries
+			reg := regexp.MustCompile(`server: .*`)
+			kubeConfigContents, err := helper.ReadFile(path)
+			Expect(err).To(BeNil())
+			kubeConfigContents = reg.ReplaceAllString(kubeConfigContents, "server: https://not-reachable.com:443")
+
+			// write to a new file which will be used as the new kubeconfig
+			newKubeConfigPath := filepath.Join(commonVar.Context, "newKUBECONFIG")
```

Comment:
Why not use the `WithEnv` approach that was introduced in #4472? https://github.com/openshift/odo/blob/ea12bdf96c4b99b256bcb1aea7d5477afb900683/tests/helper/helper_cmd_wrapper.go#L41

Reply:
Already using it at https://github.com/openshift/odo/pull/4508/files#diff-298ac2fa009dfd61614b93eb6f167f458a022631741170d39af9aa9b8aac16f9R148

Reply:
OK. After our discussion on Slack, I think I get why you're doing this. It's about specifically setting an incorrect value in kubeconfig.